packages/ckeditor5-dev-env/lib/release-tools/changelog/writer-options.js

```diff
@@ -153,3 +155,15 @@ function getCommitType( commit ) {
 			throw new Error( `Given invalid type of commit ("${ commit }").` );
 	}
 }
+
+function getPackageJson() {
```

Comment:
This is a premature optimisation. The performance of this tool (which is absolutely not important because it's a dev tool) would not even be affected in any significant manner by removing this caching because the OS/disk will cache this file itself.

KISS.